code/modules/events/grid_check.dm

```diff
@@ -1,11 +1,20 @@
-/datum/event/grid_check	//NOTE: Times are measured in master controller ticks!
-	announceWhen		= 5
+#define ANNOUNCE_DELAYED FALSE
+#define ANNOUNCE_NOW TRUE
+
+#define ANNOUNCE_AFTER_MC_TICKS 5
+#define APC_TOGGLE_OFF_PROBABILITY 25
+#define EVENT_FIXES_ITSELF_AT_END FALSE
+#define EVENT_MAX_LENGTH_MC_TICKS 150
+#define EVENT_MIN_LENGTH_MC_TICKS 60
```

Comment:
These should be variables on the event object, not defines. They should also have less ambiguous naming, even though they are `#undef`'d at the EOF.

Reply:
I have made some changes here. Please review and resolve if the changes are OK?